src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/ViewDataDictionaryFactory.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace Microsoft.AspNetCore.Mvc.ViewFeatures.Internal
+{
+    public class ViewDataDictionaryFactory
+    {
+        public static ViewDataDictionary CreateViewDataDictionary(IModelMetadataProvider metadataProvider)
```

Comment:
Should I add a work item for changing all references to the internal ctor to the other constructor with ModelStateDictionary? There are many other tests apart from the ones impacted by IVTs which use it.